nodeadm/api/v1alpha1/nodeconfig_types.go

```diff
@@ -129,10 +129,16 @@ const (
 )
 
 // Feature specifies which feature gate should be toggled
-// +kubebuilder:validation:Enum={InstanceIdNodeName}
+// +kubebuilder:validation:Enum={InstanceIdNodeName,AggressiveImagePullTradeoff}
```

Comment:
Can we call this `AggressiveImagePull`? Not a hill I need to die on if you think `Tradeoff` is adding something notable

Reply:
Done, no strong preference.